### ISSUE TITLE
loopd exited with an error: timeout: couldn't obtain exclusive lock on loop.db

### ISSUE THREAD

**Issue by mrmcbitcoin** [lightninglabs/loop#502]
I had an issue after my desktop froze while I was running loopd manually from the CLI.

2022-06-03 13:18:55.003 [INF] LOOPD: Version: 0.18.0-beta commit=
2022-06-03 13:18:55.004 [INF] LNDC: Creating lnd connection to localhost:10009
2022-06-03 13:18:55.007 [INF] LNDC: Connected to lnd
2022-06-03 13:18:55.017 [INF] LNDC: Waiting for lnd to unlock
2022-06-03 13:18:55.135 [INF] LNDC: Wallet state of lnd is now: Lnd main server is ready for requests
2022-06-03 13:18:55.306 [INF] LNDC: lnd version: v0.14.3-beta, build tags 'autopilotrpc,signrpc,walletrpc,chainrpc,invoicesrpc,watchtowerrpc,monitoring,kvdb_postgres,kvdb_etcd'
2022-06-03 13:18:55.307 [INF] LNDC: Using network mainnet
2022-06-03 13:18:55.308 [INF] LNDC: Waiting for lnd to be fully synced to its chain backend, this might take a while
2022-06-03 13:18:55.392 [INF] LNDC: lnd is now fully synced to its chain backend
2022-06-03 13:18:55.393 [INF] LOOPD: Swap server address: swap.lightning.today:11010

loopd exited with an error: timeout: couldn't obtain exclusive lock on /home/zeusman/.loop/mainnet/loop.db, timed out after 5s: make sure no other loop daemon process (standalone or embedded in lightning-terminal) is running

**Comment by mrmcbitcoin** [comment]
I have this solved but I wanted to make a ticket so other people can find my solution.

First I checked to make sure no other loopd or LITD process was running.

To fix this I did:

cd .loop/mainnet/
chmod 500 loop.db

then when I tried to run loopd again:
loopd exited with an error: open /home/zeusman/.loop/mainnet/loop.db: permission denied

finally I did:
chmod 600 loop.db

Now its running again.
